src/meili/match_route.cc

```diff
@@ -73,7 +73,7 @@ EdgeSegment::EdgeSegment(baldr::GraphId the_edgeid,
 
 /**
  * Here we return the vector of edge segments between the source and target states. If its a node to
- * node route (meaning no realy edge is traversed) then we use the target_result to say what edge the
+ * node route (meaning no relay edge is traversed) then we use the target_result to say what edge the
```

Comment:
this is why I hate stuff like this. this is wrong and this tool will surely make other mistakes. personally I can live with imperfection for the sake of not living under the boot of another annoying CI bot. if we put this in CI it should not fail the build

Reply:
I did actually review all of the changes and they made sense to me. "realy" isn't a word as far as I can tell and "relay edge" somewhat makes sense to me. What would be the correct fix, or what does "realy" mean?

Reply:
it's supposed to be 'real'

Reply:
it's because you don't know the structure of our graph. this edge is fake because it has no spatial component. it connects a node to a copy of itself on another level of the graph. our graph is divided into levels where we can separate the edges by their importance. this allows for search space pruning via something akin to data partitioning